tests/support/case.py

```diff
@@ -46,18 +45,27 @@
 log = logging.getLogger(__name__)
 
 
-class ShellTestCase(TestCase, AdaptedConfigurationTestCaseMixin, ScriptPathMixin):
+class ShellCase(TestCase, AdaptedConfigurationTestCaseMixin, ScriptPathMixin):
     """
     Execute a test for a shell command
     """
 
-    def run_salt(self, arg_str, with_retcode=False, catch_stderr=False, timeout=15):
+    RUN_TIMEOUT = 30
```

Comment:
Looks like we are changing timeouts from 60 to 30. Most of these long timeouts were added to address flakiness in the windows test suite.

Reply:
Even before we started skipping slow tests, these passed on windows. This is just a rebase againsr latest changes.
If we do get some failures on windows, we can just increase those and for windows alone giving us yet another chance to identify slow tests on windows.